client/jetpack-cloud/sections/agency-dashboard/sites-overview/types.ts

```diff
@@ -96,7 +96,9 @@ export interface Site {
 	site_stats: SiteStats;
 	onSelect?: () => void;
 	jetpack_boost_scores: BoostData;
-	php_version_num: number;
+	php_version_num: string;
```

Comment:
The `php_version_num` field is a number returned by the endpoint

Reply:
Reverted back to number in fb88090.

Reply:
Reverted back to number in fb88090.